### ISSUE TITLE
Show new category for highest rated routes

### ISSUE THREAD

**Issue by Wotuu** [Wotuu/keystone.guru#803]
Now we'll have to see if this does not overload the /routes pages etc, but it'd be nice to have it (it was requested before already).

**Comment by Wotuu** [comment]
#804 must be completed first for this. I can then prompt people to rate a route after completing their dungeon which should give more ratings and to make the rating page make a bit more sense.
